styx-cli/src/main/java/com/spotify/styx/cli/CliMain.java

```diff
@@ -865,6 +880,7 @@ public Subparser parser(Subparsers subCommands) {
     CREATE("", "Create a backfill"),
     EDIT("e", "Edit a backfill"),
     HALT("h", "Halt a backfill"),
+    UNHALT("u", "Unhalt a halted backfill"),
```

Comment:
Could we call this `resume` instead? 😄

Reply:
`resume` is not technically/strictly antonym of `halt`. :D 

This is hard, and it was hard when choosing `halt` as I remember.